### PR TITLE
Updates to Inundation symbology

### PR DESCRIPTION
**Pull request by CHafen**
Hopefully final updates to Inundation symbology. Making a few fixes to the Thalweg symbology, fixing Arc crest type symbology which wasn't working and adding the rest of the webRave symbology

### CODE REVIEW

**Comment by CHafen** [comment]
@MattReimer I can't figure out why this pull request is failing to pass the checks. Can you advise me on what might be causing this pull request to fail the checks? Thanks

**Review by MattReimer — APPROVED** [review]

File: Symbology/web/Inundation/valley_bottom.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://xml.riverscapes.xyz/Symbology/web/vector.schema.json",
+  "legend": [
+    ["hsl(60, 100%, 50%)", "Valley Bottom"],
```
Comment (MattReimer) [inline Symbology/web/Inundation/valley_bottom.json:4]: There's an extra comma at the end of this line

**Review by MattReimer — CHANGES_REQUESTED** [review]
Really minor things. I'll approve and merge this when the automated tests pass.

File: Symbology/web/Inundation/valley_bottom.json (hunk 1 above)
Comment (CHafen) [reply Symbology/web/Inundation/valley_bottom.json]: Thanks, fixed it

**Review by MattReimer — APPROVED** [review]

**Pull request merged**